lib/Timer/src/timer.cpp

```diff
@@ -0,0 +1,16 @@
+#include "timer.h"
+
+Timer::Timer() {}
+
+void Timer::start_timer(unsigned int duration) {
+    nextTimeout = millis() + duration;
+}
+
+bool Timer::timer_has_expired() {
```

Comment:
You don't have to do this necessarily, but here is a cool one-liner for this function:
*(millis() > _nextTimeOut) ? return true : return false;*

Terrible readability I know, but nice sometimes haha :P